teamcity: Jepsen tests fail due to ssh connect timeout

The following tests appear to have failed:

[#405731](https://teamcity.cockroachdb.com/viewLog.html?buildId=405731):


```
--- FAIL: Jepsen/Jepsen: JepsenG2: JepsenG2/subcritical-skews (86.329s)
None
--- FAIL: Jepsen/Jepsen: JepsenMonotonic: JepsenMonotonic/subcritical-skews+start-kill-2 (86.115s)
None
--- FAIL: Jepsen/Jepsen: JepsenSequential: JepsenSequential/subcritical-skews+start-kill-2 (609.970s)
None
--- FAIL: Jepsen/Jepsen: JepsenSequential: JepsenSequential/majority-ring+start-kill-2 (0.327s)
None
--- FAIL: Jepsen/Jepsen: JepsenSequential: JepsenSequential/parts+start-kill-2 (0.318s)
None
--- FAIL: Jepsen/Jepsen: JepsenSets: JepsenSets/majority-ring (0.315s)
None
--- FAIL: Jepsen/Jepsen: JepsenSets: JepsenSets/split (0.320s)
None
--- FAIL: Jepsen/Jepsen: JepsenSets: JepsenSets/start-kill-2 (0.324s)
None
--- FAIL: Jepsen/Jepsen: JepsenSets: JepsenSets/start-stop-2 (0.311s)
None
--- FAIL: Jepsen/Jepsen: JepsenSets: JepsenSets/strobe-skews (0.310s)
None
--- FAIL: Jepsen/Jepsen: JepsenSets: JepsenSets/subcritical-skews (0.321s)
None
--- FAIL: Jepsen/Jepsen: JepsenSets: JepsenSets/majority-ring+subcritical-skews (0.317s)
None
--- FAIL: Jepsen/Jepsen: JepsenSets: JepsenSets/subcritical-skews+start-kill-2 (0.317s)
None
--- FAIL: Jepsen/Jepsen: JepsenSets: JepsenSets/majority-ring+start-kill-2 (0.324s)
None
--- FAIL: Jepsen/Jepsen: JepsenSets: JepsenSets/parts+start-kill-2 (0.325s)
None

```

Please assign, take a look and update the issue accordingly.

--- a/build/jepsen/terraform/main.tf
+++ b/build/jepsen/terraform/main.tf
@@ -178,6 +178,9 @@ resource "null_resource" "cockroach-runner" {
       "sudo apt-get -qqy upgrade -o Dpkg::Options::='--force-confold' >/dev/null",
       # Allow access to the cockroach instances from the Jepsen controller.
       "sudo cp ~/.ssh/authorized_keys2 /root/.ssh/authorized_keys2",
+      # The Jepsen controller does a lot of separate ssh commands which can
+      # trigger sshguard. Disable it.
+      "sudo service sshguard stop",
       # Download cockroach binary, zip so that Jepsen understands it
       "mkdir -p /tmp/cockroach",
       "[ $(stat --format=%s cockroach) -ne 0 ] || curl -sfSL https://edge-binaries.cockroachdb.com/cockroach/cockroach.linux-gnu-amd64.${var.cockroach_sha} -o cockroach",

--- a/build/teamcity-jepsen-run-one.sh
+++ b/build/teamcity-jepsen-run-one.sh
@@ -84,8 +84,7 @@ if timeout 20m ssh "${SSH_OPTIONS[@]}" "ubuntu@${controller}" "${testcmd}" \
 
     # Test passed. grab just the results file.
     progress "Test passed. Grabbing minimal logs..."
-    # TODO(bdarnell): remove -v's
-    scp "${SSH_OPTIONS[@]}" -C -r -v -v -v \
+    scp "${SSH_OPTIONS[@]}" -C -r \
         "ubuntu@${controller}:jepsen/cockroachdb/store/latest/{test.fressian,results.edn,latency-quantiles.png,latency-raw.png,rate.png}" \
         "${artifacts_dir}"
 

